frontend-sdk/src/lib/state/PasswordState.ts

```diff
@@ -0,0 +1,69 @@
+import { State } from "./State";
+import { UserState } from "./UserState";
+
+/**
+ * @interface
+ * @category SDK
+ * @subcategory Internal
+ * @property {number=} retryAfter - Timestamp until when the next password login can be attempted in seconds (since January 1, 1970 00:00:00 UTC).
```

Comment:
```suggestion
 * @property {number=} retryAfter - Timestamp (in seconds since January 1, 1970 00:00:00 UTC) indicating when the next password login can be attempted.
```